### ISSUE TITLE
And what would it be like for macOS?

### ISSUE THREAD

**Issue by DruChill** [Ruddernation-Designs/Adobe-URL-Block-List#20]
Help mi bro Im student :c

**Comment by Ruddernation-Designs** [comment]
I've not tried tbh, but I do have a Mac I can try it on and let you know

**Comment by DruChill** [comment]
It's okay, I hope it works, I'll be waiting.

**Comment by Nygosaki** [comment]
I am using sequia, the hosts file is located at `/private/etc/hosts`. Photoshop completely refuses to even open

**Comment by Ruddernation-Designs** [comment]
I still can't get my Mac to use the list, but I do have it all blocked as I'm using the PC's internet as an access point

**Comment by Nygosaki** [comment]
> I still can't get my Mac to use the list, but I do have it all blocked as I'm using the PC's internet as an access point

The mac version just seems to be using different servers all together IMO.   
A more reliable way to prevent the non-genuine screen on Mac seems to be using LuLu or another custom firewall and completely blocking anything going out of Adobe apps.

**Comment by DruChill** [comment]
You're right, LULU is a good blocker but even so I've already blocked everything and I get a message saying I don't remember anymore but I still sigh and get an error message from phtoshop. I checked and blocked everything.

**Comment by Nygosaki** [comment]
@DruChill I would recommend first fully uninstalling all adobe applications with something like handbreak, then reinstalling from a trusted source (which for me was appstorrent ru) and then actively blocking anything as the requests pop up during installation.

**Comment by DruChill** [comment]
So you don't get any alerts while using Photoshop?

**Comment by Oneashenzzx** [comment]
adobe软件的防弹窗：在clash verge添加规则
    - DOMAIN-SUFFIX,adobe.io,REJECT
    - DOMAIN-SUFFIX,adobe.com,REJECT
    - DOMAIN-SUFFIX,adobestats.io,REJECT
    - DOMAIN,lcs-cops.adobe.io,DIRECT

但是可能半个月、一个月后还是会弹窗，重启后会好
